### PR TITLE
Create Artist & Album view for music library

### PR DESCRIPTION
**Pull request by 1hitsong**
**Changes**
Added Artist and Album sorting to the music library, in addition to the default view. Artist view is a list of all known artists in JF as found using MusicArtist item type. Album view is a list of all known albums in JF as found using MusicAlbum item type.

**Issues**
N/A. This is a New Feature.

### CODE REVIEW

**Comment by candry7731** [comment]
1. Default view is the artist view, I would remove the "default" option
2. I would add "songs" view option

**Comment by 1hitsong** [comment]
> 1. Default view is the artist view, I would remove the "default" option

It's actually a little bit different. The default view displays everything that is a child of the root library. For example, in my library I mostly have artists, but I also have several albums of songs that aren't under an artist. These albums appear in the default view, but under Artist view, they're not shown because they have no artist value.

Perhaps we can change "Default" to something more descriptive to explain the different. Any ideas? 

Root? Folder? Base?

> 2. I would add "songs" view option

In an effort to keep these PRs smaller than the last, I'm limiting how much I change. Song view will be its own separate change after this.

**Comment by candry7731** [comment]
> 
I think "Default" is fine. Thanks for the clarification. I didn't notice the no artist albums

**Review by candry7731 — APPROVED** [review]
Functions as intended.

File: components/ItemGrid/ItemGrid.brs [hunk 1]
```diff
@@ -101,9 +106,20 @@ sub loadInitialItems()
     else if m.top.parentItem.collectionType = "tvshows"
         m.loadItemsTask.itemType = "Series"
     else if m.top.parentItem.collectionType = "music"
-        m.loadItemsTask.itemType = "MusicArtist,MusicAlbum"
-        m.loadItemsTask.fallbackType = "MusicAlbum"
```
Comment (jimdogx) [inline components/ItemGrid/ItemGrid.brs:105]: I might be missing it, but I think this is the only place you were setting FallbackType.  If it's no longer being used, can you remove it elsewhere?

**Review by jimdogx — CHANGES_REQUESTED** [review]

File: components/ItemGrid/ItemGrid.brs (hunk 1 above)
Comment (1hitsong) [reply components/ItemGrid/ItemGrid.brs:105]: :+1: PR Updated

**Review by jimdogx — APPROVED** [review]

**Comment by lakerssuperman** [comment]
Humble user here again.  I just updated my Roku with the latest master code, but the artist/album sort ability seems to be gone.  The only option under view is "music" and it seems to be a mix of albums with some artists scattered throughout that have multiple albums.  Fully possible I'm doing something wrong or looking in the wrong place.  Just wanted to clarify that this PR merged the aritst/album sorting into master.  Thanks guys!!

**Pull request merged**